web/themes/custom/bevan/templates/layout/page.html.twig

```diff
@@ -23,31 +28,27 @@
       {% endif %}
 
       {% if page.sidebar_first %}
-        <div class="govuk-grid-column-one-third">
-          <aside role="complementary">
-            {{ page.sidebar_first }}
-          </aside>
-        </div>
+        <aside class="govuk-grid-column-one-third">
```

Comment:
Previously, in the default twig page template file this `<aside>` and the one for `page.sidebar_second` had a ` role="complementary"` parameter, but they don't now. Is this role parameter not needed any more? For example, that `role="complementary"` parameter has been retained in the new `page--directory` template, cf. lines 14, 45.

Reply:
Yes that's right, after my research implementing the aria roles for the directory page I think we should not add aria roles generically, there should only be one of each role per page. We shouldn't assume that either/both sidebars should have that role unless we know what content is in there.